pkg/charts/additionalchart.go

```diff
@@ -66,12 +71,14 @@ func (c *AdditionalChart) RevertMainChanges(pkgFs billy.Filesystem) error {
 		return fmt.Errorf("Working directory %s has not been prepared yet", c.WorkingDir)
 	}
 	if c.CRDChartOptions == nil {
+		// return if the additional chart is not a CRD chart
 		return nil
 	}
 	mainChartWorkingDir, err := c.getMainChartWorkingDir(pkgFs)
 	if err != nil {
 		return fmt.Errorf("Encountered error while trying to get the main chart's working directory: %s", err)
 	}
+	// copy CRD files from packages/<package>/charts-crd/crd-manifest/ to packages/<package>/charts/crds/
```

Comment:
Should we put it in `packages/<package>/charts/crds/` or `packages/<package>/charts/files/`? I'd prefer the latter since I think the `crd` directory is a special directory in Helm, so I'm worried that adding something there could create a future condition that might break... v.s. putting arbitrary files in `files/` is generally a standard practice iirc.